### DISCUSSION TITLE
Lighthouse Deck sensor damaged

### DISCUSSION THREAD

**Discussion by kleikemper** [bitcraze/discussions#1493] [Q&A]
After a crash, one of the sensors on a lighthouse looks damaged. The deck is not providing a positioning signal (LED is red permanently). Is there any way to fix it?
![cf_damaged_sensor](https://github.com/user-attachments/assets/8fb9c5f7-8dd0-4a19-9bdf-facc08a7779b)

**Comment by knmcguire** [comment]
ah no that looks quite bad and I don't think this can be fixed without actually replacing the image sensor itself.

On the product specifications you can check which IR receivers we use on the lighthouse deck so you can order those and replace them yourself, or use a different lighthouse deck. https://www.bitcraze.io/products/lighthouse-positioning-deck/
